Dockerfile

```
@@ -60,4 +61,6 @@ FROM amd64/ubuntu:bionic@sha256:2aeed98f2fa91c365730dc5d70d18e95e8d53ad4f1bbf426
 #--------------------------------------
 FROM base as final
 
-USER 1000
+ARG USER_ID
```

```suggestion
ARG USER_ID=1000
```
needs to be repeated